packages/apps/demo/src/CitySearchBar.tsx

```diff
@@ -172,6 +172,7 @@ function createSortedCityOptions(
   }
 
   return cities
+    .filter(city => countryCodeToName.get(city.properties.countryCode))
```

Comment:
Tiny nit: I have a slight preference for `.filter` predicates returning booleans, instead of values that can be coerced to booleans.
```suggestion
    .filter(city => countryCodeToName.has(city.properties.countryCode))
```

Reply:
Makes perfect sense. I appreciate you pointing this stuff out! The last time I did serious JavaScript was like 15 years ago. I'm still marveling at the `class` keyword, we used to do prototypes and worry about Internet Exploder 5.5 (cue grandpa story time 😅). Takes a while to get familiar with the modern language.

Reply:
I, too, have worked with JS for a while 😂 . The language (and the ecosystem around the language) has indeed come a long way since the Internet Exploder years!